src/backend/tests/test-data/tasks.test-data.ts

```diff
@@ -0,0 +1,44 @@
+import { Task as sharedTask } from 'shared';
+import { batman, greenlantern, wonderwoman } from './users.test-data';
+import { TaskPriority, TaskStatus } from 'shared';
+import { Prisma } from '@prisma/client';
+import taskQueryArgs from '../../src/prisma-query-args/tasks.query-args';
+import { prismaWbsElement1 } from './wbs-element.test-data';
+import userTransformer from '../../src/transformers/user.transformer';
+
+export const save_the_day: Prisma.TaskGetPayload<typeof taskQueryArgs> = {
```

Comment:
underscores are for python!

also `task` should be present in the variable name to make things understandable

`taskSaveTheDay`